src/algorithms/tonal/key.cpp

```diff
@@ -367,21 +367,21 @@ void Key::compute() {
 
 
   if (maxMajor > maxMinor && maxMajor > maxOther) {
-    keyIndex = (int) (keyIndexMajor *  12 / pcpsize + 0.5);
+    keyIndex = std::roundl(keyIndexMajor *  12 / pcpsize);
     scale = MAJOR;
     max = maxMajor;
     max2 = max2Major;
   }
 
   else if (maxMinor >= maxMajor && maxMinor >= maxOther) {
-    keyIndex = (int) (keyIndexMinor * 12 / pcpsize + 0.5);
+    keyIndex = std::roundl(keyIndexMinor * 12 / pcpsize);
     scale = MINOR;
     max = maxMinor;
     max2 = max2Minor;
     }
 
 	else if (maxOther > maxMajor && maxOther > maxMinor) {
-    keyIndex = (int) (keyIndexOther * 12 / pcpsize + 0.5);
+    keyIndex = std::roundl(keyIndexOther * 12 / pcpsize);
     scale = MAJMIN;
     max = maxOther;
     max2 = max2Other;
```

Comment:
We commonly use `round` in other places. Why using `roundl` instead?

Reply:
I was thinking about std::lround...
Will correct this.